src/views/Profile.vue

```diff
@@ -1,55 +1,94 @@
 <template>
   <v-container>
     <v-responsive>
-      Welcome {{ user.name }}
-      <div v-if="user.isAdmin">
-        <p>Search a student's archives:</p>
+      <div class="text-h5 mt-5">Welcome {{ user.userName }}</div>
+      <div v-if="user.isAdmin" class="pt-5">
+        <p class="pb-4">Search a user's archives:</p>
         <v-row>
           <v-col>
-            <v-text-field 
+            <v-text-field
+            v-model="searchQuery"
             clearable
-            placeholder="Datahub folder link:  gs://ucb-datahub-archived-homedirs/..."
+            placeholder="Please enter a Datahub Archive Link"
+            id="archive-searchbox"
           /> 
         
           </v-col>
           <v-col cols="2">
             <v-btn
-            @click="getUserUrls()"
+            class="mt-2 ml-10"
+            @click="fetchSingleUrl"
           >
           Search
           </v-btn>
```

Comment:
@cesarvh - Hartsfield should probably use ESLint for front-end code linting.  There are indentation problems above and that problem could very quickly become pervasive. Here's an example of configuring ESLint: https://github.com/ets-berkeley-edu/damien/blob/dev/.eslintrc.js